### DISCUSSION TITLE
Welcome to ServerlessWP Discussions!

### DISCUSSION THREAD

**Discussion by mitchmac** [mitchmac/ServerlessWP#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ramavats** [comment]
I am really thankful to you for creating this.

**Reply by mitchmac** [reply]
Thank you for the kind words @ramavats -- welcome to the community!

**Comment by notio-ca** [comment]
Very nice & basic, had an instant success to deploy to Vercel. But I am confused, my deployment setting are for Node.js 18, I also see an index.js as entry point. Wondering where the php processor is, and also if we can downgrade to 7.4 with vercel-php@0.3.3 (https://github.com/vercel-community/php#-versions)

Thanks!

**Reply by mitchmac** [reply]
Hi @notio-ca I'm sorry it has taken me so long to reply.

This starter project uses an npm package that I have created for the specific needs of the project (to support as many Lambda based serverless function providers and WordPress). The package name is [serverlesswp](https://www.npmjs.com/package/serverlesswp) and the repository is at [mitchmac/serverlesswp-node](https://github.com/mitchmac/serverlesswp-node).

There you can find PHP getting compiled using this [Dockerfile](https://github.com/mitchmac/serverlesswp-node/blob/main/build/Dockerfile) where the version is currently hardcoded to the most recent 8.1.* when I get around to updating it. 

You could try adapting that, or possibly integrate [@libphp/amazon-linux-2-v74](https://www.npmjs.com/package/@libphp/amazon-linux-2-v74) which is what vercel-php@0.3.3 uses under the hood.

Is it a theme or plugin that requires 7.4? I was hoping PHP 8.1 would cover most cases!

**Comment by Xczer** [comment]
i am unable to install theme from the wordpress admin panel... and vercel dont allow me to upload my theme which exceeds there size cap i wonder how can i install my theme... a annoying popup is displayed to update my wordpress version!! any fix to this would be appreciated
anyways apart from these ,nice idea keep growing!

**Reply by ramavats** [reply]
If you want to install additional themes then you will have to go to your github repo and go to wp/wp-content/themes and then you can upload the whole folder

**Reply by Xczer** [reply]
Dhanyavaad🙏

**Comment by Riddlcal** [comment]
I installed Wordpress, but after that I got a 504 error and I can't quite figure out why this is happening. The website displays fine, but I can't login because it seems it doesn't know I'm a user, probably because of the 504 error.

**Reply by notio-ca** [reply]
Set the debug to true in the wp-config.php. then you will see the error
instead of a 504 error code

**Comment by danilocasassa** [comment]
How can I add some plugin on my wordpress? 
When I'm trying do it in the wp-admin the option "add new" is not enabled.

**Reply by ramavats** [reply]
In your github repo go to `wp/wp-content/plugins` and upload the extracted folder of your preferred plugin

**Comment by HatimDa** [comment]
I currently have a WordPress site on DigitalOcean. I am interested in migrating it, but I'm unsure how to get the MySQL database working with this setup. 

I noticed that this project uses PlanetScale, but I'm uncertain if there's a way to migrate from MySQL to PlanetScale, or if it's even feasible. 

I would like to work with a serverless setup because this way I can manage all my files on my computer. I make changes then simply upload the new code cahnges to GitHub. I appreciate the ease of use this offers, but first, I need to understand the database part to facilitate the migration process.

I hope you guys can provide some guidance on this issue. 

THANKS.
